app/js/components/orderEntry/LabOrderDetails.jsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const LabOrderDetails = ({ urgency, orderer }) => (
+const LabOrderDetails = ({ urgency, orderer, activeDates }) => (
   <div className="order-details">
     <div className="details">
       <ul>
         <b>Urgency:</b> {urgency}
       </ul>
       <br />
+      <ul>
+        <b>Active Dates:</b> {activeDates}
```

Comment:
is this present in the mockup ?

Reply:
David requested for it to be present there. See here https://issues.openmrs.org/browse/OEUI-280